### DISCUSSION TITLE
Styles not loading without baseUrl in config

### DISCUSSION THREAD

**Discussion by RtYkk** [upptime/upptime#114] [Q&A] [answered]
I make a website 5 minutes ago. But it looks abnormal
![chrome_wSbIhJj9AY](https://user-images.githubusercontent.com/45971730/101790194-26818b80-3b3d-11eb-897d-8a05ca2b436c.png)
Could you please tell me how can I solve this? Thank you!
repo url: https://github.com/RtYkk/Pt-Sites-Observer
page url: https://rtykk.github.io/Pt-Sites-Observer/

**Comment by tronyx** [comment, marked as answer]
Because you're supposed to set it to publish to the root of your GitHub Pages, like this:

https://sharex.tronflix.app/zORI8/kiZoRunO64.png

If you don't, I believe you need to set this in the main YAML file, as it states:

```
# Uncomment the following line if you don't have a custom domain and add your repo name
  # baseUrl: /upptime
```

**Comment by RtYkk** [comment]
@christronyxyocum Thank you very much. But I can't access the photo.
And I just changed the mentioned line. It seems to be useless.
https://github.com/RtYkk/Pt-Sites-Observer/blob/39ea4918988ac47830fe32f1ffa5d5c6cd167041/.upptimerc.yml#L23-L24

**Comment by tronyx** [comment]
You should clear your cache because your site looks perfectly fine to me now. You also had to wait for the GitHub Actions to finish generating the data and publish it.

**Comment by RtYkk** [comment]
My fault. I use Ctrl+R. Now it looks normal for me. I really appreciate your assistance.
